spec/cypress/integration/schools/MultipleSchools.feature

```diff
@@ -0,0 +1,27 @@
+Feature: School leaders should be able to manage multiple schools
+
+  Background:
+    Given scenario "schools/multiple_schools" has been run
+    And I am logged in as existing user with email "school-leader@example.com"
+
+  Scenario: Selecting and changing schools
+    Given I am on "schools" page
+    Then "page body" should contain "Test School 1"
+    And "page body" should contain "Test School 2"
+    And the page should be accessible
+    And percy should be sent snapshot called "select school page"
```

Comment:
snapshot name seems a bit vague, maybe worth adding "induction coordinator" to it?

Reply:
Ah I thought it got prefixed with the scenario name. Apparently not